Formula/guile.rb

```diff
@@ -34,6 +35,13 @@ class Guile < Formula
 
   uses_from_macos "gperf"
 
+  # This patch fixes an issue where Guile >= 3.0.6 doesn't properly load dynamic
+  # libraries on macOS.
```

Comment:
Sorry, one more thing I forgot from last time: do we know when we can remove this patch? If so, please update the comment with this information.

Reply:
Done, thank you!

Reply:
Should be available in next release (I assume Guile 3.0.8)

Reply:
comment added.